ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/Quantity.java

```diff
@@ -48,6 +48,10 @@ public static String create(final long value) {
     return uint256ToHex(UInt256.of(value));
   }
 
+  public static String create(final byte[] value) {
+    return create(new BigInteger(1, value));
+  }
+
   public static String create(final BigInteger value) {
     return uint256ToHex(UInt256.of(value));
```

Comment:
(Optional) Looks like this class could use a bit of refactoring while we're here.  

We should get rid of `create(final BigInteger value)` and replace usages with the more direct conversion in `format(final BigInteger input)`.  We should also make the method names consistent - either name them all `create` or `format`.